src/UWP/Package.appxmanifest

```diff
@@ -16,7 +16,8 @@
   <Applications>
     <Application Id="App" Executable="$targetnametoken$.exe" EntryPoint="Bit.UWP.App">
       <uap:VisualElements DisplayName="bitwarden" Square150x150Logo="Assets\Square150x150Logo.png" Square44x44Logo="Assets\Square44x44Logo.png" Description="bitwarden" BackgroundColor="transparent">
-        <uap:DefaultTile Wide310x150Logo="Assets\Wide310x150Logo.png" ShortName="Bitwarden">
+        <uap:LockScreen Notification="badge" BadgeLogo="Assets\BadgeLogo.png" />
+        <uap:DefaultTile Wide310x150Logo="Assets\Wide310x150Logo.png" ShortName="Bitwarden" Square310x310Logo="Assets\LargeTile.png" Square71x71Logo="Assets\SmallTile.png">
```

Comment:
`bitwarden` should be all lowercase.

Reply:
Ok, should be easy to find/replace

Reply:
I only see this one line as needing an update.